scipy/linalg/_matfuncs.py

```diff
@@ -208,25 +211,42 @@ def logm(A, disp=True):
 
 
 def expm(A):
-    """
-    Compute the matrix exponential using Pade approximation.
+    """Compute the matrix exponential of an array.
 
     Parameters
     ----------
-    A : (N, N) array_like or sparse matrix
```

Comment:
Does this change not break backwards compatibility?

Reply:
Ideally people should be using `scipy.sparse.linalg.expm` and dense version kind of hijacks the sparse version.